### ISSUE TITLE
missing documentation

### ISSUE THREAD

**Issue by MellianStudios** [cartalyst/sentinel#521]
Hello, while working with reminder I noticed method exists no longer returns instance like in v2 but just bool and there is method get (which have to be found by looking into ReminderRepositoryInterface). Would be great for future package users to have it in documentation.

**Comment by brunogaspar** [comment]
> I noticed method exists no longer returns instance like in v2 but just bool

Yes, this is correct, is documented https://cartalyst.com/manual/sentinel/3.x#reminder-exists

> and there is method get (which have to be found by looking into ReminderRepositoryInterface)

You're more than welcome to add it to the documentation by sending a pull request :)
